<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameHistoryMove</class>
 <widget class="QWidget" name="GameHistoryMove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2,1">
   <item>
    <widget class="QLabel" name="lbl_TurnId">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_XMove">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_DetectiveMove">
     <property name="text">
      <string>Detective</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_XLocation">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
